scripts/utils/options.ts

```diff
@@ -147,6 +147,9 @@ export function createReplaceData(opts: BuildOptions): Record<string, any> {
     '__VERSION:TERSER__': opts.terserVersion,
     '__VERSION:TYPESCRIPT__': opts.typescriptVersion,
 
+    // this is a hack
```

Comment:
Maybe worth extending on the docs here and mentioned why it is needed or pointing to a comment made in other files.

Reply:
that was just an oversight actually, we don't need to do anything in the `createReplaceData` function